src/langcheck/augment/en/_gender/_gender_pronouns.py

```diff
@@ -0,0 +1,137 @@
+from dataclasses import dataclass
+from types import MappingProxyType
+
+
+@dataclass(frozen=True)
+class _BaseGenderPronouns:
+    subject: str
+    dependent_possessive: str
+    object: str
+    independent_possessive: str
+    reflexive: str
+
+
+_FEMALE_PRONOUNS = _BaseGenderPronouns(subject='she',
+                                       dependent_possessive='her',
+                                       object='her',
+                                       independent_possessive='hers',
+                                       reflexive='herself')
+
+_MALE_PRONOUNS = _BaseGenderPronouns(subject='he',
+                                     dependent_possessive='his',
+                                     object='him',
+                                     independent_possessive='his',
+                                     reflexive='himself')
+
+_PLURAL_PRONOUNS = _BaseGenderPronouns(
+    subject='they',
+    dependent_possessive='their',
+    object='them',
+    independent_possessive='theirs',
+    reflexive='themselves',
+)
+
+# Neutral pronouns list were taken from some of the most popular neutral
+# pronouns from the following sources:
+# https://ofm.wa.gov/sites/default/files/public/shr/Diversity/BRGs/Pronouns%20Compilation%20Best%20Practices.pdf
+# https://uwm.edu/lgbtrc/support/gender-pronouns/
+_NEO_PRONOUNS_LIST = [
```

Comment:
It is surprising that there are so many gender-neutral pronouns!!

Reply:
So do I, and there seems more not-well-known ones.